beacon/validator/errors.go

```diff
@@ -23,21 +23,12 @@ package validator
 import "github.com/berachain/beacon-kit/errors"
 
 var (
-
-	// ErrNilPayload is an error for when there is no payload
-	// in a beacon block.
-	ErrNilPayload = errors.New("nil payload in beacon block")
```

Comment:
not used anymore since we do not check in validators anymore that payload is not nil.